modules/tensor_mechanics/src/userobjects/AbaqusUExternalDB.C

```diff
@@ -37,8 +37,11 @@ AbaqusUExternalDB::AbaqusUExternalDB(const InputParameters & parameters)
     _plugin(getParam<FileName>("plugin")),
     _library(_plugin + std::string("-") + QUOTE(METHOD) + ".plugin"),
     _uexternaldb(_library.getFunction<uexternaldb_t>("uexternaldb_")),
-    _aqSTEP(getParam<int>("abaqus_step")),
-    _current_execute_on_flag(_fe_problem.getCurrentExecuteOnFlag())
+    _aqSTEP(0),
+    _current_execute_on_flag(_fe_problem.getCurrentExecuteOnFlag()),
+    _step_user_object(isParamSetByUser("step_user_object")
+                          ? &getUserObject<StepUserObject>("step_user_object")
```

Comment:
Again, automatic would be my preference. And this might have to go into inotialSetup